demo/config.js

```diff
@@ -127,7 +127,12 @@ shakaDemo.Config = class {
         .addBoolInput_(MessageIds.DELAY_LICENSE,
             'drm.delayLicenseRequestUntilPlayed')
         .addBoolInput_(MessageIds.LOG_LICENSE_EXCHANGE,
-            'drm.logLicenseExchange');
+            'drm.logLicenseExchange')
+        .addNumberInput_(MessageIds.UPDATE_EXPIRATION_TIME,
+            'drm.updateExpirationTime',
+            /* canBeDecimal= */ false,
```

Comment:
I think this could reasonably be a non-integer number.

Reply:
I changed it!